### DISCUSSION TITLE
Fully static website taking up Edge Requests

### DISCUSSION THREAD

**Discussion by spaceemotion** [vercel/community#6640] [Help] [answered]
### Summary

I got an email about edge requests being used up. I have a static website using Astro. Since April 11th we used up over 500k requests (while we do get a lot of traffic, it never was a lot bandwidth).

There should be no calculation happening at all. Why is that the case? I don't understand how a static website can take up edge resources. Thanks in advance!

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/429147/860d2a84-220c-499c-b5bf-9e608926552c)
![image](https://github.com/vercel/community/assets/429147/aca3c16f-dfcf-4a44-bbc9-c160bd247bb2)

**Comment by northeastdev** [comment]
I have the same issue. Last month one of my project got 106 clicks in 30 days, but somehow there are 12000 edge requests in the last 30 days. And the other project which received 300K clicks has made over 2.2M edge requests. Both the sites are static and cached and follows the optimizations suggested by Vercel mentioned here - [https://vercel.com/docs/pricing/networking](https://vercel.com/docs/pricing/networking). I don't know how this is possible.

**Comment by molebox** [comment, marked as answer]
Hi! When users make requests to your site (i.e. They visit your site) you incur an Edge Request at the closest region to the visitor. These two docs help explain:
**Definition**: https://vercel.com/docs/pricing/networking#edge-requests
**Real world example**: https://vercel.com/docs/pricing/how-does-vercel-calculate-usage-of-resources

Let me know if you have any questions!

**Reply by molebox** [reply]
Directly related to your question on how a static site can incur this usage, from the docs:
> The browser requests the page from Vercel. Since it's static and cached on our global [Edge Network](https://vercel.com/docs/edge-network/overview), this only involves [Edge Requests](https://vercel.com/docs/pricing/networking#edge-requests) (the network requests required to get the content of the page) and [Fast Data Transfer](https://vercel.com/docs/pricing/networking#fast-data-transfer) (the amount of content sent back to the browser).

**Reply by Huibq** [reply]
The current rules make projects deployed on Vercel extremely vulnerable. Even a small amount of Challenge Collapsar can exhaust the number of Edge Requests. Even if it’s not under attack, if the website has a slightly larger number of clicks, it can consume all the Edge Requests in a few days. This is completely different from the experience before the modification.

**Reply by z4nr34l** [reply]
As I tested in last days - enabling Cloudflare proxy is making this count 0.7 times higher just for being enabled. I've 2 apps - bigger with traffic is making 2.5 times less edge requests count then 2nd with no traffic at all. This metric is going crazy even on very, very similar projects :D

**Reply by z4nr34l** [reply]
Managed to get deeper into that - for anyone with such problem:

- Disable Cloudflare proxy (if there is any - it's amplifying requests count by x0,7)
- Disable multiple monitoring solutions, run only one
- Switch them to make HEAD requests instead of GET
- Make your monitoring solutions check intervals longer
- Host images/files from Cloudflare R2 behind proxy

I've reduced requests on test application from ~2,5k/h to under 100/h (25x less!)

**Reply by leerob** [reply]
@z4nr34l Thanks for sharing back. You can also use [Vercel Blob for image/file storage](https://vercel.com/docs/storage/vercel-blob).

**Reply by z4nr34l** [reply]
@leerob yeah, that's right - but Vercel Blob is still missing custom domains functionality even on Pro tier.

**Comment by rico-et22** [comment]
I have the same problem too.
One of my projects (mostly SSG/ISR) has a target user base of circa 1,000-1,500, yet also since April 11 more than 1 million edge requests have been invoked that had never been invoked before. Enabling the Attack Challenge Mode doesn't seem to lower the count.
<img width="931" alt="image" src="https://github.com/vercel/community/assets/28688352/2759856c-6677-4e00-b97a-9d316b7f077f">

**Reply by Saxofounix** [reply]
@rico-et22 Were you able to track this down? I strangely enough also see a huge spike around the same time, with no apparent reason.

**Reply by rico-et22** [reply]
@Saxofounix Not yet, Cloudflare proxy is off, I'm not using Vercel Analytics, ISR revalidate time is 12h, so basically everything from [this comment](https://github.com/orgs/vercel/discussions/6640#discussioncomment-9259960) is fulfilled but still the edge requests count is very big

**Reply by prajjwal2-3** [reply]
I also faced the same issue, it has used , here are the stats
![image](https://github.com/vercel/community/assets/121849327/5d026fe2-4806-4a17-83e9-8c44f6d9b197)
It has paused all of my deployment. can someone help me with that @leerob @rico-et22

**Comment by edurguti** [comment]
something changed... I am running a  hobby project and using nextjs image optimization (my images are hosted on aws s3 bucket) I am seeing a lot of EdgeRequets to the _next/image path:

<img width="241" alt="image" src="https://github.com/user-attachments/assets/3fff2d38-55ac-4526-ab87-681218486627">
How do I avoid this?

each image I have on my website counts as en EdgeRequest?

[a comment by an automated account is not shown]
